packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -28,6 +28,7 @@ export const SubscriptionRedirect = ({
   return (
     <div className="subscription-ready">
       <h2>Your subscription is ready</h2>
+      <div className="graphic graphic-fpn"></div>
```

Comment:
Also, if you're going to go ahead and compress that SVG, you can take out the `graphic` class here too since we're currently only styling `graphic-fpn`.

Reply:
`.graphic` is inherited from the content server styles